src/MusicalScore/Graphical/VexFlow/VexFlowStaffEntry.ts

```diff
@@ -27,8 +27,17 @@ export class VexFlowStaffEntry extends GraphicalStaffEntry {
         for (const gve of this.graphicalVoiceEntries as VexFlowVoiceEntry[]) {
             if (gve.vfStaveNote) {
                 gve.vfStaveNote.setStave(stave);
-                gve.applyBordersFromVexflow();
-                this.PositionAndShape.RelativePosition.x = gve.vfStaveNote.getBoundingBox().x / unitInPixels;
+                try {
+                    gve.applyBordersFromVexflow();
+                    this.PositionAndShape.RelativePosition.x = gve.vfStaveNote.getBoundingBox().x / unitInPixels;
```

Comment:
I really really really don't like try catch if not necessary. And I think you can acutally get the isformatted information from vexflow (stavenote.js):
```javascript
getBoundingBox() {
if (!this.preFormatted) {
   throw new Vex.RERR('UnformattedNote', "Can't call getBoundingBox on an unformatted note.");
 }
```

Reply:
ok, i'll see if i can get the information from VexFlow. But maybe sometimes you can't avoid VexFlow throwing errors.

Reply:
done (b54eb2588c5a0c178caf54288b975065671c2359). this adds a dependency on how VexFlow is implemented, but it looks cleaner.